csi/build.rs

```diff
@@ -5,4 +5,13 @@ fn main() {
         .build_server(true)
         .compile(&["proto/csi.proto"], &["proto"])
         .unwrap_or_else(|e| panic!("csi protobuf compilation failed: {}", e));
+    tonic_build::configure()
+        .build_server(true)
+        .compile(&["proto/mayastornodeplugin.proto"], &["proto"])
+        .unwrap_or_else(|e| {
+            panic!(
+                "mayastor node grpc service protobuf compilation failed: {}",
+                e
+            )
```

Comment:
should use expect() here

Reply:
I'll leave it as it is, as we use panic! elsewhere.

Reply:
I suspect because it's printing e rather than just an expect message

Reply:
I suspect because it's printing e rather than just an expect message.

Reply:
This is from the definition of expect
`pub fn expect(self, msg: &str) -> T
Returns the contained Ok value, consuming the self value.
Panics if the value is an Err, with a panic message including the passed message, and the content of the Err.`

So expect() *will* generate a panic with exactly what we want: a message that includes the value passed to expect() followed by the error message. In fact the above is exactly equivalent to
`.expect("mayastor node grpc service protobuf compilation failed")`

Reply:
Done